web/src/models/DataStore.model.ts

```diff
@@ -1,4 +1,15 @@
-import {SupportedDataStores, TDataStore, TRawDataStore} from 'types/Config.types';
+import {SupportedDataStores} from 'types/Config.types';
+import {Model, TDataStoreSchemas} from 'types/Common.types';
+
+export type RawDataStore = TDataStoreSchemas['DataStore'];
+type DataStore = Model<
```

Comment:
Define the type as the same name as the default exported function but do not exported